### PR TITLE
Adjusts coin prices

### LINKED ISSUE #41927: Coin press bypasses cargo credit values
The coin press found on the mining outpost can create a total number of coins with a higher total value than if the materials were sold via the cargo shuttle.
Minting coins also ignores the 'stock' value drop that would occur had it been sold raw.

Each sheet has enough material for 5 coins
Based on the raw export values from /datum/export/material/ here's a quick chart to explain things.
- bananium = 1000, coin = 1000,  *5 = 5000
- diamond   = 500,   coin = 500,    *5 = 2500
- plasma      = 200,   coin = 100,    *5 = 500
- uranium    = 100,   coin = 80,      *5 = 400
- gold          = 125,   coin = 50,      *5 = 250
- silver         = 50,     coin = 20,      *5 = 100
- metal        = 5,       coin = 1,        *5 = 5

Since coins can be directly added to your account using your ID, miners can effectively act alone and directly buy their own goods, especially by mining out the clown ruin's bananium and minting it all for tens of thousands of credits (2 sheets = 10k credits), which you can then use to just directly buy all your needs and get swat gear and guns without busting the crates open or having actual security access.

You can even create a cash loop by buying plasteel and minting it into coins, however the process is tedious since for whatever reason you can't sell coins via the cargo shuttle or dump them into the vault ATM.

Adamantine also can't be minted even though the coin exists and it's legitimately possible to get it now via xenobiology contrary to #19068, so xenobiologists hoping to make even more cash with adamantine are out of luck here.
> it would be trivial to add it back if when it would become a true material.

Maybe just remove the coin press for now, perhaps have it as a station project in the future or have it moved to Centcom, which IMO sounds better than tweaking coin values, the material used for each coin, or reverting the ability to directly add hard currency to your ID.
...
> I don't know where Nanotrasen got this coin press from, or by what authority, if any, they have to be making actual currency, or more especially why we have it. All I can say is this thing sure wasn't made by the techs at RnD, and that for everything myself and the crew have seen we still don't have answers, but what's the point trying to get them if you'll just disappear?

### PR DESCRIPTION
**Pull request by MMMiracles**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: MMMiracles
balance: You can no longer crash the market with coins.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

Resolves #41927

### CODE REVIEW

**Comment by coiax** [comment]
Meh. A coin is just a 1/5 of a metal bar, I think, so just have the sales price actually calculated properly for the mineral coins?

**Comment by nicbn** [comment]
Or remove the coin press from mining.

**Comment by MMMiracles** [comment]
Would I be better off making a define with each mineral's value and having both the coin/ore pull from that or having the coin somehow pull from the relevant ore item itself?

**Comment by ExcessiveUseOfCobblestone** [comment]
Define sounds saner

File: code/modules/mining/ores_coins.dm [hunk 1]
```diff
@@ -382,37 +382,37 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
...
+	value = 200 //makes the clown cry
 	materials = list(MAT_BANANIUM = MINERAL_MATERIAL_AMOUNT*0.2)
 	grind_results = list("banana" = 4)
 
 /obj/item/coin/adamantine
 	name = "adamantine coin"
 	cmineral = "adamantine"
 	icon_state = "coin_adamantine_heads"
-	value = 1500
+	value = 100
```
Comment (yoyobatty) [inline code/modules/mining/ores_coins.dm]: shouldn't this be more? the original price is 500 more than diamond

Comment (MMMiracles) [reply code/modules/mining/ores_coins.dm]: 1/5th of an individual sheet's value

Comment (AlexTheSysop42) [reply code/modules/mining/ores_coins.dm]: adamantine was supposed to be incredibly valuable, it was nerfed when bounties were added

**Comment by Shadowflame909** [comment]
Remember Cargo Bounties nerfed the prices of Ores. Which is why no one sells plasma anymore. So if this seems a bit jarring, that's why.

Just a change that got through and under everyone's noses.

**Comment by Cyprex** [comment]
Note that this will severely reduce the value if roundstart/looted coins, such as the ones found in maintenance, making them pretty much useless. Due to this I'm against this PR.

**Comment by MMMiracles** [comment]
people downvoting making coins have same export values because ???

**Comment by Tlaltecuhtli** [comment]
just remove flexibility for exported minerals tbh, diamonds being less worth than a corgi skin after you sell 5 of them is dumb and having cargo sell (((excess))) minerals is interacting and interesting gameplay

**Comment by Tlaltecuhtli** [comment]
just remove flexibility for exported minerals tbh, diamonds being less worth than a corgi skin after you sell 5 of them is dumb and having cargo sell (((excess))) minerals is interacting and interesting gameplay

**Comment by AlexTheSysop42** [comment]
@MMMiracles because the bounty pr stealth nerfed a lot of ore prices and bounties arent comparable, people are only noticing this now, and your pr just cements the nerfs
I'd recommend changing ore prices to what they originally were and removing k_elasticity from most bars (except adamantine, which is somewhat easy to spam lategame and get MAD DOSH), then nerfing coins as needed

also unrelated but why is techwebs the whole-station upgrade system wasn't that supposed to be cargo or something
maybe techwebs was introduced because cargo was hobbled so much it couldn't get anything useful

**Comment by Cyprex** [comment]
The whole point of the mineral nerfs was to force you to rely on bounties for cargo points. However with the economy credits became ubiquitious again. It ultimately depends on where you want cargo to go.

**Comment by AlexTheSysop42** [comment]
And the problem with that is that bounties aren't good enough to rely on for everything you might need to buy

**Comment by zamolxius** [comment]
this is whiney and overly pedantic

**Comment by optimumtact** [comment]
@AlexTheSysop42 it's because if we left the "upgrade-thestation' thing in a single department it gets controlled by that department (just see how unpopular the protolathes and techwebs change were overall)

**Pull request merged**